supadata/__init__.py

```
@@ -3,18 +3,23 @@
-    Scrape,
-    Map,
     CrawlJob,
     CrawlPage,
-    CrawlResponse
+    CrawlResponse,
+    Map,
+    Scrape,
+    Transcript,
+    TranscriptChunk,
+    TranslatedTranscript,
+    YoutubeChannel,
```

Ruff (0.8.2)

`supadata.types.YoutubeChannel` imported but unused; consider removing, adding to `__all__`, or using a redundant alias

(F401)